### PR TITLE
Add badge and optimize travis time

### PR DESCRIPTION
**Pull request by hunterlxt**
Signed-off-by: TXXT <hunterlxt@live.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 [![Crates.io](https://img.shields.io/crates/v/grpcio.svg?maxAge=2592000)](https://crates.io/crates/grpcio)
 [![docs.rs](https://docs.rs/grpcio/badge.svg)](https://docs.rs/grpcio)
+[![](https://github.com/tikv/grpc-rs/workflows/CI/badge.svg)](https://github.com/tikv/grpc-rs/actions)
 [![Build Status](https://travis-ci.org/pingcap/grpc-rs.svg)](https://travis-ci.org/pingcap/grpc-rs)
```
Comment (BusyJay) [inline README.md:8]: Can be changed to "tikv/grpc-rs".

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 [![Crates.io](https://img.shields.io/crates/v/grpcio.svg?maxAge=2592000)](https://crates.io/crates/grpcio)
 [![docs.rs](https://docs.rs/grpcio/badge.svg)](https://docs.rs/grpcio)
+[![](https://github.com/tikv/grpc-rs/workflows/CI/badge.svg)](https://github.com/tikv/grpc-rs/actions)
```
Comment (BusyJay) [inline README.md:7]: Missing alt text.

File: benchmark/README.md [hunk 3]
```diff
@@ -6,13 +6,13 @@ Quick Start
...
 ```
-$ git clone https://github.com/pingcap/grpc-rs.git
+$ git clone https://github.com/tikv/grpc-rs.git
 ```
 
 2. Clone grpc
 
 ```
-$ git clone https://github.com/pingcap/grpc.git
+$ git clone https://github.com/tikv/grpc.git
```
Comment (BusyJay) [inline benchmark/README.md:15]: Actually it should be pingcap.

**Comment by hunterlxt** [comment]
I noticed that `ping_pong::test_insecure` failed sometimes today

**Pull request merged**